console-ui/src/design/portal.css

```
@@ -2787,6 +2790,21 @@ body {
+  margin: 0.35rem 0 0.75rem;
+  padding: 0.55rem 0.75rem;
+  border: 1px solid var(--warn-border, #e7c88a);
+  border-radius: 8px;
+  background: var(--warn-bg, #fdf3e3);
+}
+.auto-fail-tail {
+  margin: 0.45rem 0 0;
+  max-height: 9rem;
+  overflow: auto;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)